chained-actions/intro.md

```diff
@@ -0,0 +1,12 @@
+# Apache OpenWhisk on OpenShift
+
+Apache OpenWhisk allows chaining of actions.  We will now create a simple sequence of actions which will split, convert to uppercase, and
```

Comment:
Change "Apache OpenWhisk allows chaining of actions.  We" to "In this scenario,".